magpylib/_lib/classes/magnets.py

```diff
@@ -283,4 +283,4 @@ def getB(self, pos):
 
 
 def __repr__(self):
-    return "name: {} \n magnetization: x: {}mt, y: {}mt, z: {}mt \n dimensions: x: {}mm, y: {}mm, z: {}mm \n position: x: {}mm, y:{}mm, z: {} \n axis: {}".format(*self.name, *self.magnetization, *self.dimension, *self.position, *self.axis)
\ No newline at end of file
+    return "name: {} \n magnetization: x: {}mt, y: {}mt, z: {}mt \n dimensions: x: {}mm, y: {}mm, z: {}mm \n position: x: {}mm, y:{}mm, z: {} \n axis: x: {}, y: {}, z:{}".format(*self.name, *self.magnetization, *self.dimension, *self.position, *self.axis)
```

Comment:
Please add the docstring with an example for this method.

Also, change `mt` to `mT` which is [mili Tesla](https://en.wikipedia.org/wiki/Tesla_(unit)) unit, and add a space between the angle and the degree string.